### ISSUE TITLE
Remote Software Engineering  Best Practices article

### ISSUE THREAD

**Issue by rinkug** [betterscientificsoftware/bssw.io#855]
EB member: @markcmiller86 

I will be happy to write this if voted in:
https://cacm.acm.org/opinion/articles/252174-the-10-best-practices-for-remote-software-engineering/

**Comment by markcmiller86** [comment]
I like it.

On the topic of productivity there, I do worry that we in the software world are too quick to come to conclusions along the lines that *producitivity is not easily measured*. While I do agree that so much of what a good software developer does is not immediately measurable by simple metrics we've reached for in the past, I do think we need to do a better job of explaining how and why productivity measures that have worked (maybe they haven't worked as well as I think) in other human labor endeavors do or do not work in software development and whether we can be more creative in adapting that research. 

I view much of my effort during any given day of software development as less about my own personal productivity and more about what I do to contribute to other's productivity...VisIt users who are encountering problems, other developers who need a PR reviewed or need a draft PR assessed for whether the direction being taken (e.g. design) is a good one, explaing to other developer/colleagues some new Git feature I've learned about, etc.
